### PR TITLE
CPP-642 migrate from circleci/* to cimg/*

### PR DESCRIPTION
**Pull request by serena97**
These images will be EOL from Dec 31st, and we should migrate to the modern cimg/* equivalents.

### CODE REVIEW

**Comment by serena97** [comment]
automatically closed 🤖

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,12 +6,7 @@ references:
   container_config_node12: &container_config_node12
     working_directory: ~/project/build
     docker:
-      - image: circleci/node:12-browsers
-
-  container_config_lambda_node12: &container_config_lambda_node12
```
Comment (serena97) [inline .circleci/config.yml:11]: I removed this `container_config_lambda_node12` because it's not being used in the config.yml

**Review by leannecornish-ft — APPROVED** [review]
Looks good to me!

**Comment by serena97** [comment]
FYI the old node-sass and n-foundations-ui version were using python2 and the new docker circleci img only had python3 not python2 so I updated these dependencies to a version that is using python3

**Pull request merged**